reference/api/search.md

```diff
@@ -435,21 +444,21 @@ To get only the `overview` and `title` fields, set `attributesToRetrieve` to `["
 **Expected value**: an array of attribute or `["*"]`
 **Default value**: `null`
 
-Crops the selected attributes' values in the returned results to the length indicated by the [`cropLength`](#crop-length) parameter.
+Crops the selected fields to the length indicated by the [`cropLength`](#crop-length) parameter.
 
-When this parameter is set, a field called `_formatted` will be added to `hits`. The cropped version of each document will be available there.
+By default, crop boundaries are marked by the ellipsis character (`…`). You can change this by using the [`cropMarker`](#crop-marker) search parameter.
 
-Optionally, you can indicate a custom crop length for any of the listed attributes: `attributesToCrop=["attributeNameA:25", "attributeNameB:150"]`. The custom crop length set in this way has priority over the `cropLength` parameter.
+When `attributesToCrop` is set, each returned document contains an extra field called `_formatted`. This object contains the cropped version of the selected attributes.
 
-Instead of supplying individual `attributes`, you can provide `["*"]` as a value: `attributesToCrop=["*"]`. This will crop the values of all attributes present in [`attributesToRetrieve`](#attributes-to-retrieve).
+Optionally, you can indicate a custom crop length for any attributes given to `attributesToCrop`: `attributesToCrop=["attributeNameA:5", "attributeNameB:9"]`. If configured, these values have priority over `cropLength`.
 
-**Cropping starts at the first occurrence of the search query**. It only keeps `cropLength` characters on each side of the first match, rounded to match word boundaries.
+Instead of supplying individual attributes, you can provide `["*"]` as a wildcard: `attributesToCrop=["*"]`. This causes `_formatted` to include the cropped values of all attributes present in [`attributesToRetrieve`](#attributes-to-retrieve).
 
-If no query word is present in the cropped field, the crop will start from the first word.
+**Cropping starts at the first occurrence of any one of the terms present in the search query.** If Meilisearch does not find any query terms in a field, cropping begins at the first word.
```

Comment:
Should this section mention(spec, lines 245-253):

>Attributes configured in `attributesToCrop` are cropped even if not present in the `searchableAttributes` index setting.
Attributes defined in the `stopWords` index setting are counted as words regarding `cropLength`.
Tokenizer separators aren't counted as words regarding `cropLength`.

Reply:
I'm trying to keep info specific to the length of a crop and what affects it on `cropLength` parameter, whilst keeping this one more about general cropping behaviour. Do you think that works?